### DISCUSSION TITLE
Changing Default Activity After Switching To Another Activity in Java

### DISCUSSION THREAD

**Discussion by floatbar** [community/community#78294] [Programming Help]
### Body

**I have two activities, which are named as "MainActivity" and "ChatActivity". The default activity within my mobile app is initially MainActivity. However, when I switch from MainActivity to ChatActivity using Intent, I want the activity "ChatActivity" to be the default activity. And besides that, when switching to MainActivity again, I want MainActivity to be the default activity. How can I do this in the MainActivity and ChatActivity classes?**

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by heysem-useinsider** [comment]
Hi, 

Just to be clear, what do you mean by wanting an activity to be the default one? do you mean that you want the app to remember the last open activity and startup with it the next time it gets launched?

Please share more details regarding your navigation structure and how you want your activities to behave.
